app/templates/application.hbs

```diff
@@ -1,16 +1,20 @@
 {{page-title "My | Real Dev Squad"}}
 
+<MobileDialog
+@dev ={{this.isDevMode}}
+@deviceType = {{this.toVisible}}
```

Comment:
- we are passing a boolean here should we change the prop name from `deviceType` to something more representative of the value being passed?